app/controllers/api/v1/questions_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
+    class QuestionsController < ApplicationController
+      before_action :authenticate_user!
+
+      # POST /api/v1/questions
+      def create
+        if params.has_key?(:subject) && params.has_key?(:body)
```

Comment:
I think you can just do `params[:subject] && params[:body]`. This way if the param is empty, it will also retrieve false

Reply:
I tried that but when the param is an empty string it returns true.

I find a solution with: <params[:subject].present? && params[:body].present?>